pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -175,7 +174,7 @@ let
   applePackage' = namePath: version: sdkName: sha256: let
     pname = builtins.head (lib.splitString "/" namePath);
     appleDerivation = appleDerivation' pname version sdkName sha256;
-    callPackage = pkgs.newScope (packages // pkgs.darwin // { inherit appleDerivation; });
+    callPackage = pkgs.newScope (packages // pkgs.darwin // { inherit appleDerivation pname version; });
```

Comment:
This change is not part of "darwin.apple-source-releases: drop sdkName in version"